src/SearchQueryService/Controllers/SearchController.cs

```diff
@@ -126,7 +128,52 @@ private static object CreateAzSearchResponse(AzPost newDocs)
         }
 
         private async Task PostDocuments(string indexName, AzPost docs)
-            => await _solrService.PostDocumentAsync(ConvertAzDocs(docs), indexName);
+        {
+            var delete = new AzPost { Value = new List<Dictionary<string, JsonElement>>() };
+            var addOrUpdate = new AzPost { Value = new List<Dictionary<string, JsonElement>>() };
+
+            foreach (var doc in docs.Value)
+            {
+                if (doc[_searchAction].GetString() == "delete")
```

Comment:
What if the value is `DELETE`? Use the `.Equel()` method instead.
```suggestion
                if (doc[_searchAction].GetString().Equals("delete", StringComparison.OrdinalIgnoreCase))
```

Reply:
This shouldn't happen - according to docs https://docs.microsoft.com/en-us/rest/api/searchservice/addupdate-or-delete-documents